### ISSUE TITLE
Crash in dabe_aw11 with single attribute policy in python 2.7

### ISSUE THREAD

**Issue by mhlakhani** [JHUISI/charm#56]
I downloaded and installed the python2.7 tarball from here: https://github.com/downloads/JHUISI/charm/Charm-Crypto-0.42-Python2.7.tar.gz

In the dabe_aw11 file, there is a sample test that runs successfully. However, it fails if the policy is changed to just have one attribute. The following gist contains the test case that fails:
https://gist.github.com/mhlakhani/a012a85aa8c47ce37eed

In particular, the error is below:

```
Traceback (most recent call last):
  File "test3.py", line 17, in <module>
    decrypted_msg = dabe.decrypt(public_parameters, secret_keys, cipher_text)
  File "/usr/local/lib/python2.7/dist-packages/Charm_Crypto-0.42-py2.7-linux-x86_64.egg/charm/schemes/dabe_aw11.py", line 154, in decrypt
    egg_s *= ( (num / dem) ** coeffs[x] )
pairing.Error: unsupported right operand types: int, bytes, str
```

This same test case works without crashing in the python3 version as far as I can tell.

**Comment by jakinyele** [comment]
Thanks. We'll fix this asap.

**Comment by jakinyele** [comment]
Fixed in 0.43
